dotcom-rendering/src/components/EditionSwitcherBanner.importable.tsx

```diff
@@ -29,6 +29,11 @@ const content = css`
 	align-items: flex-start;
 	${center}
 
+	svg {
+		fill: ${palette.brand[400]};
```

Comment:
It's not obvious in the Storybook docs, but the icons all have a `theme` prop that you could use to override the colour instead. eg. `theme={{ fill: palette.brand[400] }}`